printf.c

```diff
@@ -912,3 +914,12 @@ int fctprintf(void (*out)(char character, void* arg), void* arg, const char* for
   va_end(va);
   return ret;
 }
+

```

Comment:
Please do not modify the source for this project, but rather, create a separate file (say, `fuzz_printf.c`) to implement your libFuzzer target.